pkg/minishift/openshift/registry.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openshift
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	instanceState "github.com/minishift/minishift/pkg/minishift/config"
+)
+
+func init() {
+	systemKubeConfigPath = filepath.Join(constants.Minipath, "machines", constants.MachineName+"_kubeconfig")
```

Comment:
@LalatenduMohanty Since both are part of openshift package then better to go with using `systemKubeConfigPath` instead again creating same variable for in this file. Also if you want to remove init method then you also need to change the variable name otherwise it will conflict with name which is already in package.  Below changes works without any init method.

```
var systemKubeConfig = filepath.Join(constants.Minipath, "machines", constants.MachineName+"_kubeconfig")
cmdArgText := fmt.Sprintf("get -o jsonpath={.spec.clusterIP}:{.spec.ports[*].port} %s -n %s --config=%s", service, namespace, systemKubeConfig) 
```